homeassistant/components/electric_kiwi/manifest.json

```diff
@@ -0,0 +1,14 @@
+{
+  "domain": "electric_kiwi",
+  "name": "Electric Kiwi",
+  "config_flow": true,
+  "integration_type": "hub",
```

Comment:
Just realised - should this be `service` instead?

From [the blog post](https://developers.home-assistant.io/blog/2022/10/24/hubs-devices-services-virtual-integrations/)

> A `hub` provides a gateway to multiple other devices or services. `service` and `device` are integrations that provide a single device or service per config entry.

Reply:
You'll also need to add that to the flick electric integration :smile: